blog-posts/scoped-registry.md

```diff
@@ -0,0 +1,100 @@
+
+`@scope:registry=https://<your registry>` where `@scope` is your scope name and `<your registry>` is the registry where your scoped packages are hosted. 
+
+If you need authentication you can add an additional line below the scoped registry with `_auth` and the basic authentication string for username and password (`username:password` ⇒ base64 encoded) or `_authToken` for an access token:
+
+`//<your registry>:_auth=<base64-string>`
+
+## What is this all about?
+
+Nowadays almost every programming language comes with its own package management solution, so developers can easily share packages with other developers around the world. For Python it is pip, for .NET it`s nuget. npm is the package manager solution for Javascript based languages. It is used to create and use node packaged modules and is built into the Javascript platform [Node.js](http://www.nodejs.org/). The central component behind these package managers is a registry. A registry is a database of packages, each comprised of software and metadata. For example the public registry for npm is [registry.npmjs.org](http://registry.npmjs.org).
+
+Beside using public registries, companies can establish their own private registries in their company network. The advantage is that the published packages of that company never leave the company networks. Another advantage is that you can setup authentication to additionally secure your internally published packages.
+
+## Scopes and scoped registries
+
+If you are a javscripty/typescript developer in a company you certainly stumbled over the problem that you, on the one side, use official packages from the public npm registry but on the other side also need internal packages from the company internal registry. Unfortunately there cannot be two main registries in npm at the same time. But scoped packages come to the rescue.
+
+> All npm packages have a name. Some package names also have a scope. A scope follows the usual rules for package names (URL-safe characters, no leading dots or underscores). When used in package names, scopes are preceded by an `@` symbol and followed by a slash, e.g. `@somescope/somepackagename` Scopes are a way of grouping related packages together, and also affect a few things about the way npm treats the package.
+> -- <cite>[npm](https://docs.npmjs.com/cli/v8/using-npm/scope)</cite>
+
+
+Packages can be grouped together with scopes and for scopes we can set alternative registries named `scoped registries` where npm will lookup the packages instead of the main registry.
+
+> 💡 The usage of scoped packages and scoped registries can also greatly diminish your attack surface for the dependency confusion attack [dependency confusion attack](https://snyk.io/blog/detect-prevent-dependency-confusion-attacks-npm-supply-chain-security/)
```

Comment:
I like thos small bits of knowledge you placed 👍

Reply:
I thought it might be of interest with the recent topics and also another reason not to use a proxy repository which is possible in e.g. nexus :smile: